src/js-feature-toggle-manager.ts

```diff
@@ -39,31 +39,38 @@ export class FeatureToggleManager {
   /**
    * Provides a getter method for Feature Toggle Manager singleton object.
    */
-  static get instance(): FeatureToggleManager {
-    if (undefined === this._instance) {
-      this._instance = new FeatureToggleManager();
-      return this._instance;
+  static get Instance(): FeatureToggleManager {
+    if (undefined === this.instance) {
+      this.instance = new FeatureToggleManager();
+      return this.instance;
     }
 
-    return this._instance;
+    return this.instance;
+  }
+
+  /**
+   * Getter for available feature toggles.
+   */
+  public get Toggles(): ToggleStatus[] {
+    return this.toggles;
```

Comment:
@samiam236 there you go, that's the standard we use in C#!

Private visibility is lowercase whereas public visibility is uppercase.